<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartScaleControls</class>
 <widget class="QDockWidget" name="ChartScaleControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Power Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Minimum</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Maximum</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Span</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="minBox">
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>-30.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="maxBox">
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.989999999999995</double>
      </property>
      <property name="value">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="spanSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLCDNumber" name="spanLCD"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Units</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Center</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="unitsBox">
      <item>
       <property name="text">
        <string>Volts</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>dBm</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="autoScaleButton">
      <property name="text">
       <string>Auto-Scale</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSlider" name="centerSlider">
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLCDNumber" name="centerLCD"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spanSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spanLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centerSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>centerLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
